rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -405,6 +405,9 @@ public void registerBtcTransaction(
                 case PEGOUT_OR_MIGRATION:
```

Comment:
I am wandering, the Bridge will consider the `fundTx` as a pegout since the `fundTx` will contain a change output to the current federation? Or how does it determine that this is a pegout? Not sure if it makes sense :)

Reply:
because the sigHash was saved in the pegout index when it was created, thats how we recognize pegouts now :)
See [here](https://github.com/rsksmart/rskj/blob/56119ab694f4068c25d18125aaf2bd1f31b17f7a/rskj-core/src/main/java/co/rsk/peg/PegUtils.java#L50).

[This](https://github.com/rsksmart/rskj/blob/56119ab694f4068c25d18125aaf2bd1f31b17f7a/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L1209) other method takes care of saving all the information when requesting a release, and [we call it](https://github.com/rsksmart/rskj/blob/56119ab694f4068c25d18125aaf2bd1f31b17f7a/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L1005) after creating the fund transaction

Let me know if thats clear enough :)

Reply:
Thanks, clear enough :)